### PR TITLE
_colors.scss added

### LINKED ISSUE #459: Creating a colors.scss file
### Description

We need to create a colors.scss file that will hold scss variables derived from the chosen color palette.

### 👮‍♀️ Reviewer

Ali Başaran

### ⏰ Deadline

24.10.2023

### PR DESCRIPTION
**Pull request by alperen-bircak**
closes #459 . Example use has been added to the mainlayout component.

### CODE REVIEW

File: app/frontend/src/_colors.scss [hunk 1]
```diff
@@ -0,0 +1,183 @@
+// Vanilla Colors
```
Comment (cisel-zumbul) [inline app/frontend/src/_colors.scss:1]: Can you add a comment here, stating which shade is darker, lighter. Like is color-10 darker than color-20?

**Comment by AliBasarann** [comment]
The colors are great and appropriate to the palette we chose. Yet, I cannot see the gray color, can you add it?

**Comment by alperen-bircak** [comment]
I've incorporated both of your requests into this branch. Once we've reviewed the changes, we can proceed with merging it.

**Comment by AliBasarann** [comment]
I saw no problem. I am merging.

**Pull request merged**